src/Typeahead/HighlightingOption.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export const HighlightingOption = ({ label, position, value }) => {
```

Comment:
I'm not sure what you mean here.  I don't think there are any appropriate default values and the PropTypes section below indicate that the three params are required.

What was your thinking on this?